otherlibs/stdune/src/stdune.ml

```diff
@@ -30,6 +30,32 @@ module Pp = struct
   let compare ~compare x y =
     Ordering.of_int (Pp.compare (fun a b -> Ordering.to_int (compare a b)) x y)
   ;;
+
+  let to_dyn tag_to_dyn t =
```

Comment:
I've added Pp.to_dyn since I've been wanting this for a while and it make our life easier when testing.